open-rosa/src/main/java/org/odk/collect/openrosa/parse/OpenRosaResponseParser.kt

```diff
@@ -8,5 +8,5 @@ import org.odk.collect.openrosa.forms.EntityIntegrity
 interface OpenRosaResponseParser {
     fun parseFormList(document: Document): List<FormListItem>?
     fun parseManifest(document: Document): List<MediaFile>?
-    fun parseIntegrityResponse(doc: Document): List<EntityIntegrity>?
+    fun parseIntegrityResponse(doc: Document): List<EntityIntegrity>
```

Comment:
We still want to return `null` in case where document is invalid so form download fails